src/main/java/com/cloudogu/smeagol/wiki/domain/Wiki.java

```diff
@@ -0,0 +1,67 @@
+package com.cloudogu.smeagol.wiki.domain;
+
+import com.google.common.base.MoreObjects;
+
+import java.net.URL;
+import java.util.Objects;
+
+public class Wiki {
+
+    private final WikiId id;
+    private final URL repositoryUrl;
+    private final DisplayName displayName;
+    private final Path directory;
+    private final Path landingPage;
+
+    public Wiki(WikiId id, URL repositoryUrl, DisplayName displayName, Path directory, Path landingPage) {
+        this.id = id;
+        this.repositoryUrl = repositoryUrl;
+        this.displayName = displayName;
+        this.directory = directory;
+        this.landingPage = landingPage;
+    }
+
+    public WikiId getId() {
+        return id;
+    }
+
+    public URL getRepositoryUrl() {
+        return repositoryUrl;
+    }
+
+    public DisplayName getDisplayName() {
+        return displayName;
+    }
+
+    public Path getDirectory() {
+        return directory;
+    }
+
+    public Path getLandingPage() {
+        return landingPage;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS00122)